tutorials/darcy_thermo_mech/doc/content/workshop/problem/step01.md

```diff
@@ -37,7 +37,7 @@ A basic MOOSE input file requires six parts, each of which will be covered in gr
 - `[Kernels]`: Define the equation(s) to solve
 - `[BCs]`: Define the boundary condition(s) of the problem
 - `[Executioner]`: Define how the problem will be solved
-- `[Outputs]`: Define how the solution will be written
+- `[Outputs]`: Define how the solution will be returned
```

Comment:
I honestly like `written` better. `returned` seems more in-code API like

Reply:
We ll switch it back next time then